packages/provider/src/HocuspocusProviderWebsocket.ts

```diff
@@ -98,8 +98,6 @@ export class HocuspocusProviderWebsocket extends EventEmitter {
     url: '',
     // @ts-ignore
     document: undefined,
-    // @ts-ignore
-    awareness: undefined,
```

Comment:
I don't think this is being used anywhere now that the websocket is separate from the provider(s).